lusidtools/cocoon/seed_sample_data.py

```diff
@@ -66,6 +67,35 @@ def seed_data(
 
         data_frame = getattr(pd, f"read_{supported_files[file_type]}")(transaction_file)
 
+    def check_or_set_default_value(mapping, check_key, default_value):
+        """
+        This function check whether the mappings variables have the required default values to be uploaded
+        via the load_from_data_frame function.
+
+        Parameters
+        ----------
+        mapping : dict
+            a file containing mapping of DataFrame headers to LUSID headers.
+
+        check_key : str
+            a string that represents the key to be checked
+
+        default_value : obj
+            an object such as an empty dictionary or list
+
+        Returns
+        -------
+        mapping : dict
+            a file containing mapping of DataFrame headers to LUSID headers.
+
+        """
+
+        for key, value in mapping.items():
```

Comment:
you can simplify this to

```
if check_key not in mapping:
    value.update({check_key: default_value})
```

Reply:
yes - you're right. will fix and re-push